ql/src/test/results/clientpositive/llap/compare_cols_null.q.out

```diff
@@ -121,6 +126,7 @@ POSTHOOK: query: select * from ccn_table where key <=> '123a'
 POSTHOOK: type: QUERY
 POSTHOOK: Input: default@ccn_table
 #### A masked pattern was here ####
+NULL	t1
```

Comment:
Why do we have this change of results with the patch?

Reply:
I think it is because we are inserting `String` values to nullable `INT` field:
`create table ccn_table(key int, value string);`
`insert into ccn_table values('123a', 't1'), ('123', 't2');`

`('123a', 't1')` is inserted as `NULL, 't1'`

And, in this select statement: `select * from ccn_table where key <=> '123a';` `key <=> '123a'` resolves to `key IS NOT DISTINCT FROM null`, which returns the output seen above.

Reply:
Yes, I think new output makes sense. However, I am surprised before this patch, the result was different?

Reply:
Yes, before this patch, the queries with `<=>` didn't return any results! Although the explain plan did return an output.